GameData/RealismOverhaul/Engine_Configs/Stentor_Config.cfg

```diff
@@ -36,7 +36,7 @@
 
 	%title = Stentor Booster
 	%manufacturer = Armstrong Siddeley
-	%description = The booster chamber of the Stentor rocket. Stentor was developed as a two chamber booster-sustainer for the Blue Steel missile, with one chamber providing boost thrust, while another smaller chamber provided sustainer thrust. The smaller chamber was developed into the Gamma series of rocket engines. The larger chamber was proposed to be used for several projects, but was ultimatley never developed further.
+	%description = Stentor was developed as a two chamber booster-sustainer for the Blue Steel missile, with one chamber providing boost thrust and another smaller chamber providing sustainer thrust. The smaller chamber was developed into the Gamma series of rocket engines. The larger chamber was proposed to be used for several projects, but was ultimately never developed further.
```

Comment:
This should probably be a bit more clear that this just represents the booster half of Stentor.